workflow.rst

```diff
@@ -856,14 +856,14 @@ Then you can access this metadata in your controller as follows::
 
 There is a ``getMetadata()`` method that works with all kinds of metadata::
 
-    // pass no arguments to getMetadata() to get "workflow metadata"
-    $title = $workflow->getMetadataStore()->getMetadata()['title'];
+    // pass a string (the key of workflow metadata) to getMetadata() to get "workflow metadata" by key
+    $title = $workflow->getMetadataStore()->getMetadata('title');
 
-    // pass a string (the place name) to getMetadata() to get "place metadata"
-    $maxNumOfWords = $workflow->getMetadataStore()->getMetadata('draft')['max_num_of_words'];
+    // pass two arguments: a string (the key of place metadata) and a string (the place name) to getMetadata() to get "place metadata" by key
+    $maxNumOfWords = $workflow->getMetadataStore()->getMetadata('max_num_of_words', 'draft');
```

Comment:
What about using `getPlaceMetadata('draft')['max_num_of_words']` instead (same for the other methods)?

Reply:
Yes, of course this, in [docs](https://symfony.com/doc/current/workflow.html#storing-metadata) ->getMetadataStore()->getMetadata() is given like alternative to those method, right? 
Also this ->getMetadataStore()->getMetadata() is used two times differently:

`// pass no arguments to getMetadata() to get "workflow metadata"
$title = $workflow->getMetadataStore()->getMetadata()['title'];`

`// $workflow is a Workflow instance retrieved from the Registry (see above)
$title = $workflow->getMetadataStore()->getMetadata('title', $transition);`

And in the trait I saw that the correct way to use getMetadata is: 


`// $workflow is a Workflow instance retrieved from the Registry (see above)
$title = $workflow->getMetadataStore()->getMetadata('title', $transition);`

Am I wrong?

Reply:
Ah sorry, you are right. I missed that this paragraph is about accessing all metadata through one single method.

Reply:
No problem. You are welcome :)